### DISCUSSION TITLE
FurAffinity Update

### DISCUSSION THREAD

**Discussion by toxicwind** [tophf/mpiv#97] [MPIV host rules]
Whatever is on the wiki doesn't work, by the way, updated one:
`{"r": "furaffinity\\.net/view/[0-9]+","q": "img#submissionImg"}`

**Comment by tophf** [comment]
Thanks.

I've added it as a scoped rule so that it works only on this site:

```json
{"d":"furaffinity.net", "e": "a[href^='/view/']",
 "q": "img#submissionImg"
}
```
